src/components/Modal/Modal.stories.tsx

```diff
@@ -11,38 +11,32 @@ import Dropdown from '../Dropdown/Dropdown';
 import { Typography } from '../Typography';
 import color from '../../styles/colors';
 
-const hasPositionAbsoluteFix = {
-    transform: 'scale(1)',
-    height: '90vh',
-};
-
 export default {
     title: '5.Popup/Modal',
     component: Modal,
-    decorators: [
```

Comment:
why do you want to remove this fix? Without it the docs are totally broken, see?

![Screenshot 2022-05-02 at 16 54 14](https://user-images.githubusercontent.com/13779759/166255743-f91c37fe-d680-4946-a74a-eef8d8ed2111.png)

Reply:
Maybe I removed it by wrong can't remember why I removed it
Will add it back